apps/api/v2/src/modules/users/users.repository.ts

```diff
@@ -83,3 +97,17 @@ export class UsersRepository {
     });
   }
 }
+
+function capitalizeTimezone(timezone: string) {
+  const segments = timezone.split("/");
+
+  const capitalizedSegments = segments.map((segment) => {
+    return capitalize(segment);
+  });
+
+  return capitalizedSegments.join("/");
+}
+
+function capitalize(str: string) {
+  return str.charAt(0).toUpperCase() + str.slice(1).toLowerCase();
+}
```

Comment:
Can we maybe create a single function `capitalize` and this would take in two params `userInput` and `userInputType` (either timeZone or normal string) and then based on our userInputType we return the modified `userInput` ? Just throwing it out there although it's most probably an overkill

Reply:
I think its fine as is